manifests/config.pp

```diff
@@ -80,6 +80,34 @@
     ensure  => absent,
   }
 
+  if $::foreman::jobs_sidekiq {
+    if $::foreman::jobs_sidekiq_redis_url == undef {
+      include ::redis
+
+      redis::instance { 'dynflow':
+        port => $::foreman::jobs_sidekiq_redis_port,
+      }
+    }
+
+    file { '/etc/foreman/dynflow/':
+      ensure  => directory,
+    }
+
+    file { '/etc/foreman/dynflow/orchestrator.yml':
+      owner   => 'root',
+      group   => $::foreman::group,
+      mode    => '0640',
+      content => template('foreman/dynflow_orchestrator.yml.erb'),
+    }
+
+    file { '/etc/foreman/dynflow/worker.yml':
```

Comment:
We are supporting one and only one worker out of the box right? So in the future this would get updated to deploy additional `worker.yml` files based on some user input?

Reply:
Yeah, exactly.